WordPress/Classes/Models/ReaderAbstractTopic+Lookup.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+
+extension ReaderAbstractTopic {
+
+    /// Fetch all `ReaderAbstractTopics` currently in Core Data.
+    ///
+    /// - Returns: An array of all `ReaderAbstractTopics` currently persisted in Core Data.
+    @objc(lookupAllInContext:error:)
+    static func lookupAll(in context: NSManagedObjectContext) throws -> [ReaderAbstractTopic] {
```

Comment:
I think TIL that you can add `error:` in the Objective-C name version of a Swift `throws` method.  Neat.